core/src/test/scala-3/cats/derived/KittensSuite.scala

```diff
@@ -16,18 +16,23 @@
 
 package cats.derived
 
+import cats.Eq
 import cats.platform.Platform
 import cats.syntax.AllSyntax
 import munit.DisciplineSuite
-import org.scalacheck.Arbitrary
+import org.scalacheck.{Arbitrary, Cogen}
 import org.scalacheck.Test.Parameters
 
+import scala.deriving.Mirror
 import scala.quoted.*
 
 /** An opinionated stack of traits to improve consistency and reduce boilerplate in Kittens tests. Note that unlike the
   * corresponding CatsSuite in the Cat project, this trait does not mix in any instances.
   */
-abstract class KittensSuite extends KittensSuite.WithoutEq, TestEqInstances
```

Comment:
What are these instances used for? I thought we were defining explicit instances for everything in `ADTS`?

Reply:
We have the `EqInstances` separately to be able to test `Eq` without interference 🐔 🥚